src/adapters/eplanning.js

```diff
@@ -0,0 +1,53 @@
+var bidfactory = require('src/bidfactory.js');
+var bidmanager = require('src/bidmanager.js');
+var adloader = require('src/adloader.js');
+
+var EPlanningAdapter = function EPlanningAdapter() {
+
+  var ISV_DEFAULT = "aklc.img.e-planning.net";
+
+  window.pbjs = window.pbjs || {};
+  window.pbjs.processEPlanningResponse = function(response) {
```

Comment:
Yes, it is on global scope as we do not use AJAX for server calling, we use JSONP. Then we need a global scoped function for catching the bid response.